### PR TITLE
Add Subnomicon Backlinks Were Relevant

### PR DESCRIPTION
**Pull request by ImmaZoni**
This PR is an attempt to send more traffic to the Subnomicon via the Docs. As such we've included many links back to the subnomicon for users easy navigation

### CODE REVIEW

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 1]
```diff
@@ -63,3 +63,12 @@ We will download two files for your respective operating system.
 <SourcePage />
 </TabItem>
 </Tabs>
+
+:::note Learn More About Syncing, Plotting & Farming
+
+If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:69]: I'm not sure whether the process here should be singular, or not 🤔 Would likely use "processes" as it refers to multiple different instances.

File: docs/farming-&-staking/farming/advanced-cli/cli-prerequisites.mdx [hunk 2]
```diff
@@ -25,6 +25,12 @@ Make sure you have a stable network connection. During the plotting phase of far
 This may impact your network usage so please check your network connection if you have a hard data limit.
 :::
 
+:::note
+
+The Subspace Networ is based on Proof of Archival Storage, as such your storage contribution will directly correlate to your rewards. For insights on how this choice impacts our network's consensus mechanism and your potential rewards, see [this guide](https://subnomicon.subspace.network/docs/consensus/overview).
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-prerequisites.mdx:30]:
```suggestion
The Subspace Network is based on Proof of Archival Storage, as such your storage contribution will directly correlate to your rewards. For insights on how this choice impacts our network's consensus mechanism and your potential rewards, see [this guide](https://subnomicon.subspace.network/docs/consensus/overview).
```

File: docs/farming-&-staking/farming/pulsar/pulsar-install.mdx [hunk 3]
```diff
@@ -49,3 +49,12 @@ Please note that some older processors/VMs are no longer supported by official r
 </TabItem>
 
 </Tabs>
+
+:::note Learn More About Syncing, Plotting & Farming
+
+If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/pulsar/pulsar-install.mdx:55]: Same comment about "process" <-> "processes"

File: docs/farming-&-staking/farming/pulsar/pulsar-prerequisites.mdx [hunk 4]
```diff
@@ -27,6 +27,12 @@ Make sure you have a stable network connection. During the plotting phase of far
 This may impact your network usage so please check your network connection if you have a hard data limit.
 :::
 
+:::note
+
+The Subspace Networ is based on Proof of Archival Storage, as such your storage contribution will directly correlate to your rewards. For insights on how this choice impacts our network's consensus mechanism and your potential rewards, see [this guide](https://subnomicon.subspace.network/docs/consensus/overview).
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/pulsar/pulsar-prerequisites.mdx:32]:
```suggestion
The Subspace Network is based on Proof of Archival Storage, as such your storage contribution will directly correlate to your rewards. For insights on how this choice impacts our network's consensus mechanism and your potential rewards, see [this guide](https://subnomicon.subspace.network/docs/consensus/overview).
```

File: docusaurus.config.js [hunk 5]
```diff
@@ -357,6 +380,14 @@ const config = {
         },
         copyright: `Copyright © ${new Date().getFullYear()} Subspace Labs, Inc.`,
       },
+      announcementBar: {
```
Comment (EmilFattakhov) [inline docusaurus.config.js:383]: Neat!

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx [hunk 6]
```diff
@@ -86,16 +86,15 @@ In general you should be able to download the latest release, and re-start the N
 
 There are some cases where version updates will cause issue with your Node & Farmer and you may have to wipe your node, typically when errors occur. If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.
 
-### Wipe & Purge
+### Wipe
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx:89]: Let's revert those changes, for 3g the preferred method of removing the data is still through `purge-chain`

Comment (EmilFattakhov) [reply versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx:89]: But, we can indeed remove the message "It does not matter if the node/farmer executable is the previous one or from the new snapshot, both will work :)" as it's no longer true.

**Review by EmilFattakhov — CHANGES_REQUESTED** [review]
Good idea to backlink Subnomicon @ImmaZoni, mostly looks good! 

Requesting a few small changes. I also think we should link [this page](https://subnomicon.subspace.network/docs/rewards_fees) from the getting started page, and add similar links to the operators/nominators part.

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx (hunk 1 above)
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:69]:
```suggestion
If you would like to learn more about how the syncing, plotting, & farming process works on the Subspace Network, please refer to our Subnomicon links below.
```

File: docs/farming-&-staking/farming/pulsar/pulsar-install.mdx (hunk 3 above)
Comment (jim-counter) [inline docs/farming-&-staking/farming/pulsar/pulsar-install.mdx:55]:
```suggestion
If you would like to learn more about how the syncing, plotting, & farming process works on the Subspace Network, please refer to our Subnomicon links below.
```

File: versioned_docs/version-latest/farming-&-staking/farming/_category_.json [hunk 7]
```diff
@@ -3,6 +3,6 @@
     "position": 3,
     "link": {
       "type": "generated-index",
-      "description": "All community provided resources are non-official. For clarification please refer to our official docs."
+      "description": "All Farming related documentation for the Subspace Network."
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/_category_.json:6]:
```suggestion
      "description": "All farming related documentation for the Subspace Network."
```

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 8]
```diff
@@ -63,3 +63,12 @@ We will download two files for your respective operating system.
 <SourcePage />
 </TabItem>
 </Tabs>
+
+:::note Learn More About Syncing, Plotting & Farming
+
+If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx:69]:
```suggestion
If you would like to learn more about how the syncing, flotting, & farming process works on the Subspace Network, please refer to our Subnomicon links below.
```

File: versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-install.mdx [hunk 9]
```diff
@@ -49,3 +49,12 @@ Please note that some older processors/VMs are no longer supported by official r
 </TabItem>
 
 </Tabs>
+
+:::note Learn More About Syncing, Plotting & Farming
+
+If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-install.mdx:55]:
```suggestion
If you would like to learn more about how the syncing, plotting, & farming process works on the Subspace Network, please refer to our Subnomicon links below.
```

File: versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-prerequisites.mdx [hunk 10]
```diff
@@ -27,6 +27,12 @@ Make sure you have a stable network connection. During the plotting phase of far
 This may impact your network usage so please check your network connection if you have a hard data limit.
 :::
 
+:::note
+
+The Subspace Networ is based on Proof of Archival Storage, as such your storage contribution will directly correlate to your rewards. For insights on how this choice impacts our network's consensus mechanism and your potential rewards, see [this guide](https://subnomicon.subspace.network/docs/consensus/overview).
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-prerequisites.mdx:32]:
```suggestion
The Subspace Network is based on Proof of Archival Storage, as such your storage contribution will directly correlate to your rewards. For insights on how this choice impacts our network's consensus mechanism and your potential rewards, see [this guide](https://subnomicon.subspace.network/docs/consensus/overview).
```

File: docs/farming-&-staking/farming/_category_.json [hunk 11]
```diff
@@ -3,6 +3,6 @@
     "position": 3,
     "link": {
       "type": "generated-index",
-      "description": "All community provided resources are non-official. For clarification please refer to our official docs."
+      "description": "All Farming related documentation for the Subspace Network."
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/_category_.json:6]:
```suggestion
      "description": "All farming related documentation for the Subspace Network."
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
A few small tweaks requested. Otherwise looks good!

File: versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-prerequisites.mdx (hunk 10 above)
Comment (ImmaZoni) [reply versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-prerequisites.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docs/farming-&-staking/farming/_category_.json (hunk 11 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/_category_.json]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-install.mdx (hunk 9 above)
Comment (ImmaZoni) [reply versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-install.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx (hunk 8 above)
Comment (ImmaZoni) [reply versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: versioned_docs/version-latest/farming-&-staking/farming/_category_.json (hunk 7 above)
Comment (ImmaZoni) [reply versioned_docs/version-latest/farming-&-staking/farming/_category_.json]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docs/farming-&-staking/farming/pulsar/pulsar-install.mdx (hunk 3 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/pulsar/pulsar-install.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx (hunk 1 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/advanced-cli/cli-install.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx (hunk 6 above)
Comment (ImmaZoni) [reply versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docusaurus.config.js (hunk 5 above)
Comment (ImmaZoni) [reply docusaurus.config.js:383]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docs/farming-&-staking/farming/pulsar/pulsar-prerequisites.mdx (hunk 4 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/pulsar/pulsar-prerequisites.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docs/farming-&-staking/farming/pulsar/pulsar-install.mdx (hunk 3 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/pulsar/pulsar-install.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docs/farming-&-staking/farming/advanced-cli/cli-prerequisites.mdx (hunk 2 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/advanced-cli/cli-prerequisites.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx (hunk 1 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/advanced-cli/cli-install.mdx]: Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 12]
```diff
@@ -66,7 +66,7 @@ We will download two files for your respective operating system.
 
 :::note Learn More About Syncing, Plotting & Farming
 
-If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
+If you would like to learn more about how the syncing, plotting, & farming processes works on the Subspace Network, please refer to our Subnomicon links below.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:69]:
```suggestion
If you would like to learn more about how the syncing, plotting and farming processes works on the Subspace Network, please refer to our Subnomicon links below.
```

File: docs/farming-&-staking/farming/pulsar/pulsar-install.mdx [hunk 13]
```diff
@@ -52,7 +52,7 @@ Please note that some older processors/VMs are no longer supported by official r
 
 :::note Learn More About Syncing, Plotting & Farming
 
-If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
+If you would like to learn more about how the syncing, plotting, & farming processes works on the Subspace Network, please refer to our Subnomicon links below.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/pulsar/pulsar-install.mdx:55]:
```suggestion
If you would like to learn more about how the syncing, plotting and farming processes works on the Subspace Network, please refer to our Subnomicon links below.
```

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 14]
```diff
@@ -66,7 +66,7 @@ We will download two files for your respective operating system.
 
 :::note Learn More About Syncing, Plotting & Farming
 
-If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
+If you would like to learn more about how the syncing, plotting, & farming processes works on the Subspace Network, please refer to our Subnomicon links below.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx:69]:
```suggestion
If you would like to learn more about how the syncing, plotting and farming processes works on the Subspace Network, please refer to our Subnomicon links below.
```

File: versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-install.mdx [hunk 15]
```diff
@@ -52,7 +52,7 @@ Please note that some older processors/VMs are no longer supported by official r
 
 :::note Learn More About Syncing, Plotting & Farming
 
-If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
+If you would like to learn more about how the syncing, plotting, & farming processes works on the Subspace Network, please refer to our Subnomicon links below.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/pulsar/pulsar-install.mdx:55]:
```suggestion
If you would like to learn more about how the syncing, plotting and farming processes works on the Subspace Network, please refer to our Subnomicon links below.
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
Missed the Oxford commas - blinded by the capitalisations! :)

LGTM after these changes.

**Comment by EmilFattakhov** [comment]
@ImmaZoni just bumping this up :) 

> I also think we should link [this page](https://subnomicon.subspace.network/docs/rewards_fees) from the getting started page, and add similar links to the operators/nominators part.

File: docs/farming-&-staking/staking/intro.md [hunk 16]
```diff
@@ -10,6 +10,10 @@ keywords:
     - Sub-domain
 ---
 
+:::note 
+This guide is focused on how to setup Operators, if you would like to learn more about the function of operators read the [Subspace Subnomicon](https://subnomicon.subspace.network/docs/category/decoupled-execution) to get a better understanding.
```
Comment (jim-counter) [inline docs/farming-&-staking/staking/intro.md:14]:
```suggestion
This guide is focused on how to setup an operator, if you would like to learn more about the function of operators read the [Subspace Subnomicon](https://subnomicon.subspace.network/docs/category/decoupled-execution) to get a better understanding.
```

File: versioned_docs/version-latest/farming-&-staking/staking/intro.md [hunk 17]
```diff
@@ -10,6 +10,10 @@ keywords:
     - Sub-domain
 ---
 
+:::note 
+This guide is focused on how to setup Operators, if you would like to learn more about the function of operators read the [Subspace Subnomicon](https://subnomicon.subspace.network/docs/category/decoupled-execution) to get a better understanding.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/staking/intro.md:14]:
```suggestion
This guide is focused on how to setup an operator, if you would like to learn more about the function of operators read the [Subspace Subnomicon](https://subnomicon.subspace.network/docs/category/decoupled-execution) to get a better understanding.
```

**Review by jim-counter — CHANGES_REQUESTED** [review]

**Pull request merged**